Sources/Internal/Keychain.swift

```diff
@@ -202,11 +202,18 @@ internal final class Keychain {
                 return SecItem.DataResult.success(Set([singleKey]))
                 
             } else if let multipleMatches = collection as? [[String: AnyHashable]] {
+                #if swift(>=4.1)
```

Comment:
Shouldn't this always build with Swift 4.0 based on the `.swift-version` file?

Reply:
If only that's how `.swift-version` worked. All that does is let Cocoapods decide that two projects are on different swift major version and therefore won't work well together.

Reply:
I thought it also affected the `SWIFT_VERSION` in the `Pods.xcodeproj` build settings?

Reply:
It may. But if you use submodules, you'll get a build warning. So we should fix this :p

Reply:
Ahh right, good point.